src/boot/efi/boot.c

```diff
@@ -1216,7 +1216,7 @@ static void config_defaults_load_from_file(Config *config, char *content) {
                         else {
                                 uint64_t u;
                                 if (!parse_number8(value, &u, NULL) || u > TIMEOUT_TYPE_MAX) {
-                                        log_error("Error parsing 'timeout' config option: %s", value);
+                                        log_warning("Cannot parse 'timeout' config option: %s", value);
```

Comment:
we usually suffix warnings messages with `, ignoring` or similar, indicating how we react with errors we don't consider fatal. Might be good to adjust these messages here at the same time as marking them as warnings